stories/button/button.stories.js

```diff
@@ -49,9 +60,9 @@ types.parameters = {
 
 export const segmentedButton = () => `
     <div class="fd-segmented-button" role="group" aria-label="Group label">
-        <button class="fd-button sap-icon--survey"></button>
-        <button class="fd-button sap-icon--pie-chart is-selected" aria-pressed="true"></button>
-        <button class="fd-button sap-icon--pool"></button>
+        <button aria-label="More" class="fd-button"><i class="sap-icon--survey fd-button__icon-left"></i></button>
+        <button aria-label="More" class="fd-button is-selected" aria-pressed="true"><i class="sap-icon--pie-chart fd-button__icon-left"></i></button>
+        <button aria-label="More" class="fd-button"><i class="sap-icon--pool fd-button__icon-left"></i></button>
```

Comment:
Rather than all having the same `aria-label` `"More"`, it should indicate the purpose of the button. You can make up ones for demo purposes based on what the icon represents e.g. "Survey", "Pie Chart", "Pool", "Cart"

Reply:
@jacobdevera Done